src/states/node.rs

```diff
@@ -1755,6 +1872,12 @@ impl Node {
         Ok(serialisation::deserialise(&serialised_connection_info)?)
     }
 
+    fn reset_rt_timer(&mut self) {
+        self.rt_msg_id = None;
```

Comment:
will there be a situation that:
rt_timer ticked so we sent out a request,
then a churn happens so we send out a section update and reset here (which cleans the rt_msg_id)
then when response comes in, we can't reorganize it?